Metro-Records/la-metro-councilmatic
# Make sort_name sortable 🙃

## Overview

This PR updates the configuration of the sort_name field so ElasticSearch will actually sort by it. Reference: https://github.com/django-haystack/django-haystack/issues/569

Connects #1142 

## Testing Instructions

This change requires an index rebuild to take effect, so you'll have to test locally.

- Run the app as instructed in the README. Make sure to follow the instructions for populating your search index.
- Run a search.
- On the results page, sort by title and confirm the sorted results load without error.
